heuristic/alns.py

```diff
@@ -301,28 +376,37 @@ def __init__(self, state, criterion, data, objective_weights, decay):
 
             remove_weighted_random_employee: [
                 repair_worst_employee_regret,
-                repair_worst_employee_greedy,
+                repair_worst_employee_greedy
             ],
 
             remove_worst_week: [
                 repair_worst_week_regret,
-                repair_worst_week_greedy,
-                repair_week_demand,
-                repair_week_demand_per_shift
+               repair_worst_week_greedy,
+                #repair_week_demand,
```

Comment:
These do not run because of incompatibility issues with no SDP-reduction?

Reply:
Yes. When updated shifts are turned off you would get a valueError